cli/medperf/__main__.py

```diff
@@ -123,53 +134,26 @@ def test(
 
 @app.callback()
 def main(
-    log: str = "INFO",
-    log_file: str = None,
-    comms: str = config.default_comms,
-    ui: str = config.default_ui,
-    host: str = config.server,
-    storage: str = config.storage,
-    prepare_timeout: int = config.prepare_timeout,
-    sanity_check_timeout: int = config.sanity_check_timeout,
-    statistics_timeout: int = config.statistics_timeout,
-    infer_timeout: int = config.infer_timeout,
-    evaluate_timeout: int = config.evaluate_timeout,
-    platform: str = config.platform,
-    cleanup: bool = True,
-    certificate: str = config.certificate,
-    local: bool = typer.Option(
-        False, help="Run the CLI with local server configuration"
-    ),
+    profile: str = typer.Option(
+        "default", help="Configuration profile to use"
+    )
```

Comment:
Currently, if a user wants to keep using their custom profile, they should keep using `--profile mycustomprofile` with every command.
I suggest the following:
- In addition to the `default` profile, we keep a reserved profile called `active`.
- We add a command: `medperf profile activate <profile_name>`. This would copy the specified profile into the reserved `active` profile.
- medperf always loads the `active` profile.

Reply:
I had implemented something similar to this, but we encounter an issue of desyncing profiles. Say I run `medperf profile active fets`. I will assume that whatever changes I do to that profile will be reflected to the active profile and viceversa. That will not be the case.

I'll see if I can make this active profile just store the name of the active profile and use that to assign the default profile.